manifests/swarms.pp

```diff
@@ -0,0 +1,4 @@
+# docker::swarms
+class docker::swarms($swarms) {
+  create_resources(docker::swarm, $swarms)
```

Comment:
Create resources in puppet > 4.0?

Reply:
What's your question?

Reply:
Iterating over a hash is much more user friendly and safe. In the least, it provides more helpful error messages